src/imports.js

```diff
@@ -4,6 +4,7 @@ import * as ReactDOM from 'react-dom';
 // API shared by Preact and React
 export const React = {
   createElement: _React_.createElement || _React_.h,
+  PropTypes: _React_.PropTypes || { oneOf: () => {} },
```

Comment:
I'll add one. Though it's clear that this is the filler if preact is used instead of react